examples/simple_operator.py

```diff
@@ -18,7 +18,7 @@
 """An example of a very simple operator on Rn."""
 
 # Imports for common Python 2/3 codebase
-from __future__ import print_function, division
+from __future__ import print_function, division, absolute_import
```

Comment:
Admittedly not the best idea to bundle with an unrelated pull request. Generally, I once in a while run the modules through the `futurize` and `pasteurize` tools from the `future` package and add (some of) the recommended changes. An obvious exception are the `unicode_literals` which can cause trouble but would still be good to have since it would make the code behave the same for both Py2 and Py3.